src/pages/cli/project/command-hooks.mdx

```diff
@@ -45,16 +45,18 @@ Minimum CLI version requirement met.
 ----- 🪝 pre-push execution end -----
 ```
 
+Please note: Hooks need to be checked into source control in order to be made available across machines. amplify pull will not re-create the hooks folder automatically as these are considered changes that need to be committed to the repository.
```

Comment:
Can we add this as an alert window under the title `Add command Hooks`

Reply:
updated